src/modules/emotionHandler/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2023 Fondazione Istitito Italiano di Tecnologia (IIT)          #
+# All Rights Reserved.                                                         #
+#                                                                              #
+################################################################################
+
+project(emotionHandler)
+cmake_minimum_required(VERSION 3.22)
```

Comment:
Is there a specific reason why you set this version? 
It is quite high, we usually set 3.12 as minimum required, unless yo don't need something specific